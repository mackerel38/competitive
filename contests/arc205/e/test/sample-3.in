12
437926 528156 284664 1038330 692060 720863 602077 1027766 868190 532252 982711 876446
