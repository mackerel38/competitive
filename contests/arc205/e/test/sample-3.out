437926
528156
284664
94842632
158208162
985930968
548875758
875494466
345599613
605424119
111381243
768586512
